ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/transaction/UtxoSignedTransaction.java

```diff
@@ -132,4 +132,9 @@ public interface UtxoSignedTransaction extends TransactionWithMetadata {
     @NotNull
     @Suspendable
     UtxoLedgerTransaction toLedgerTransaction(List<StateAndRef<?>> inputStateAndRefs, List<StateAndRef<?>> referenceStateAndRefs);
+
+    /**
+     * Verifies signatures of signatories of current {@link UtxoSignedTransaction}.
```

Comment:
The same method's doc block in `UtxoSignedTransactionInternal` had more details.